.github/workflows/code-coverage.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 
       - name: Download CSV report from previous run
         if: (github.event_name == 'schedule')
-        uses: dawidd6/action-download-artifact@v2
+        uses: dawidd6/action-download-artifact@v6
```

Comment:
Should we use official github composite action? https://github.com/actions/download-artifact, i think most of our actions use that except this

Reply:
We are downloading artifacts from a previous workflow run, [https://github.com/actions/download-artifact](https://github.com/actions/download-artifact) will need to be called differently to achieve this, we can do this later on separately.